connectors/protocol/connectors.py

```diff
@@ -556,6 +556,14 @@ async def update_last_sync_scheduled_at(self, new_ts):
             if_primary_term=self._primary_term,
         )
 
+    async def update_last_permissions_sync_scheduled_at(self, new_ts):
```

Comment:
I am wondering if we need an explicit API for this ? (especially given the very long name) as opposed to something like `connector.update(last_permissions_sync_scheduled_at=ts.isoformat())`

Reply:
> as opposed to something like `connector.update(last_permissions_sync_scheduled_at=ts.isoformat())`

One concern about exposing `update` is that you can actually update anything.

`ESDocument` and its subclasses are just data entity class (only read via `@property`), and we guard field mutating by specific methods, e.g. `heartbeat` will only allow you to update `last_seen`.